### DISCUSSION TITLE
Source code is missing from v6

### DISCUSSION THREAD

**Discussion by acd124** [aoijs/aoi.js#375] [General]
The v6 branch looks like the source code is git ignored, and only the compiled code is left. This is the opposite of what is wanted to allow for open source, since the typescript files are where the readable code is kept.

**Comment by ElTuna** [comment]
v6 isn't meant to be open sourced

**Comment by Leref** [comment]
Everything has changed in v6 and aoijs future,
At the moment the source code will not be available til further notice.
V6 isn't even complete and won't be till later summer.

**Comment by Leref** [comment]
outdated.
